cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/ListExpressionsTest.scala

```diff
@@ -120,6 +109,17 @@ RETURN
 
 A filtered list of the elements where the predicate is `true`.
 
+###assertion=returns-one parameters=value
+MATCH path = (n)-->(m)
+WHERE id(n) = %A% AND id(m) = %B%
+WITH nodes(path) AS list
+RETURN
+
+[x IN list WHERE x.prop <> $value | x.prop]
+###
+
+A list comprehension filters a list and extracts the value of the expression for each element in that list.
```

Comment:
Since both the above one also are types of list comprehensions, I'm thinking `A list comprehension that filters...` might be better. Now it can be interpreted as if all list comprehension does both filter and extract.